src/applications/vaos/new-appointment/components/CommunityCareProviderSelectionPage/ProviderSelectionField.jsx

```diff
@@ -182,13 +186,16 @@ function ProviderSelectionField({
                       <button
                         type="button"
                         onClick={() => {
-                          onChange(
-                            communityCareProviderList.find(
-                              p => p.id === checkedProvider,
-                            ),
+                          const providerIndex = communityCareProviderList.findIndex(
+                            p => p.id === checkedProvider,
                           );
+                          onChange(communityCareProviderList[providerIndex]);
```

Comment:
I think you could do `onChange(provider);` here. You already have access to the current provider because you're looping through them.

Reply:
Done